core/src/services/routing.js

```diff
@@ -148,6 +149,9 @@ export const handleRouteChange = async (
 
     const pathUrlRaw =
       path && path.length ? GenericHelpers.getPathWithoutHash(path) : '';
+    const nodeParamsRaw = pathUrlRaw.split('?')[1]
```

Comment:
I think ` const nodeParamsRaw = pathUrlRaw.split('?')[1];` would do exactly the same.